docs_user/modules/proc_adopting-image-service-with-block-storage-backend.adoc

```diff
@@ -27,8 +27,7 @@ spec
 
 .Prerequisites
 
-* Previous Adoption steps completed. Notably, MariaDB, {identity_service_first_ref} and {key_manager_first_ref}
-should be already adopted.
+* You have completed the previous adoption steps.
 
 .Procedure
 
```

Comment:
Step 1“Write the patch manifest into a file” - Does this mean "create a new file called, for example, `glance_swift.patch`, and paste the following content into it"?